src/composables/useModifierKey.ts

```diff
@@ -0,0 +1,39 @@
+import { Ref, ref } from "vue";
+import { isMac } from "@/type/preload";
+
+export const createUseKey = (key: string) => (): Ref<boolean> => {
```

Comment:
これはこのファイル名的にエクスポートされていない方が正しいかも？
```suggestion
const createUseKey = (key: string) => (): Ref<boolean> => {
```

Reply:
おっと、修正しました